### DISCUSSION TITLE
extreme high values, when restarting Home Assistant

### DISCUSSION THREAD

**Discussion by heindlalexander** [jomjol/AI-on-the-edge-device#2997] [General]
### The Problem

This is a problem, maybe only related to Home Assistant. I see no issue in my watermeter.
When I restart Home Assistant, because of installing something, I see spikes of my watermeter, which makes my energy dashboard unusable...
When restarting, the value gets unavailable in the history, and the next value is then 1004322.6 m³. See Log below, which is the export of the history data

### Version

15.7.0

### Logfile

```shell
sensor.wasserzaehler_value,68.0068,2024-03-14T17:45:07.730Z
sensor.wasserzaehler_value,68.0094,2024-03-14T17:50:07.754Z
sensor.wasserzaehler_value,68.0125,2024-03-14T17:55:07.655Z
sensor.wasserzaehler_value,68.0138,2024-03-14T18:05:07.705Z
sensor.wasserzaehler_value,unavailable,2024-03-14T18:10:12.165Z
sensor.wasserzaehler_value,1004322.6,2024-03-14T18:15:07.552Z
sensor.wasserzaehler_value,68.0198,2024-03-14T18:15:07.639Z
sensor.wasserzaehler_value,68.0199,2024-03-14T18:20:07.568Z
sensor.wasserzaehler_value,68.0213,2024-03-14T19:15:07.269Z
sensor.wasserzaehler_value,68.0331,2024-03-14T19:20:07.265Z
sensor.wasserzaehler_value,68.0424,2024-03-14T19:25:07.250Z
sensor.wasserzaehler_value,68.0450,2024-03-14T19:30:07.233Z
sensor.wasserzaehler_value,68.0461,2024-03-14T19:45:07.138Z
sensor.wasserzaehler_value,68.0462,2024-03-14T21:25:06.858Z
sensor.wasserzaehler_value,68.0472,2024-03-14T22:10:06.787Z
sensor.wasserzaehler_value,68.0511,2024-03-14T22:45:06.733Z
sensor.wasserzaehler_value,68.0526,2024-03-14T22:55:06.703Z
sensor.wasserzaehler_value,68.0600,2024-03-14T23:00:06.667Z
sensor.wasserzaehler_value,68.0601,2024-03-15T03:20:05.872Z
sensor.wasserzaehler_value,68.0640,2024-03-15T05:10:05.536Z
sensor.wasserzaehler_value,68.0645,2024-03-15T05:15:05.479Z
sensor.wasserzaehler_value,68.0649,2024-03-15T05:20:05.522Z
sensor.wasserzaehler_value,68.0728,2024-03-15T05:25:05.742Z
sensor.wasserzaehler_value,68.0748,2024-03-15T05:45:05.419Z
sensor.wasserzaehler_value,68.0754,2024-03-15T05:50:05.463Z
sensor.wasserzaehler_value,68.0764,2024-03-15T05:55:05.671Z
sensor.wasserzaehler_value,68.0785,2024-03-15T06:10:05.578Z
sensor.wasserzaehler_value,unavailable,2024-03-15T06:30:57.077Z
sensor.wasserzaehler_value,1004322.6,2024-03-15T06:35:05.422Z
sensor.wasserzaehler_value,68.0786,2024-03-15T06:35:05.461Z
sensor.wasserzaehler_value,68.0792,2024-03-15T06:40:05.350Z
sensor.wasserzaehler_value,68.0816,2024-03-15T06:45:05.403Z
```

### Screenshots

![grafik](https://github.com/jomjol/AI-on-the-edge-device/assets/91722902/904f197d-c887-4117-a195-59f5726766a6)
![grafik](https://github.com/jomjol/AI-on-the-edge-device/assets/91722902/5ed1ec09-5dee-4db3-9d47-84662e038cb9)

**Comment by caco3** [comment]
Das ist ganz komisch. Ich verwende auch HA und habe keine solchen Probleme.
Kannst Du mal schauen, was das Gerät selber für Daten aufzeichnet? (Menü Data -> Data graph)

**Reply by heindlalexander** [reply]
Hi!
Danke schonmal für die Antwort. Der Data graph sieht ok aus. Ich glaube ehrlich gesagt auch eher, dass es einen anderen Grund hat.
Vielleicht ein Verdacht: Ich hatte den Watermeter schon mal mit meinem vorherigen Wasserzähler im Einsatz, welcher dann getauscht wurde. Dieser hatte natürlich einen deutlich höheren Wert und der könnte mit den Spitzen zusammenpassen, wenn man das als Liter interpretiert. Also gut 1000 m³.
Ich habe die Werte in der DB schon gelöscht: statistics und statistics_short_term. Ich glaube aber dass er sich vielleicht irgendwo noch einen Höchstwert gemerkt hat wenn er unavailable ist, dann diesen anwendet.

**Comment by heindlalexander** [comment]
ich hoffe ich habe das Problem nun doch gelöst.
In der Zwischenzeit habe ich auch noch die Tabelle "state" bereinigt. Damit war die "History" korrekt, aber nach einem Reboot war auch hier wieder der extrem hohe Wert drinnen. Ich denke dass dieser jemals höchste Wert irgendwo noch hinterlegt ist und dieser dann angewendet wird.
Inzwischen habe ich am Watermeter dann einfach den Namen in der Konfiguration geändert, sodass der in Home-Assistant neu erscheint. Das alte Device und alle Entitäten habe ich dann auch noch aufgeräumt und nun dürfte es mit den Werten passen, auch nach einem Reboot. Ich hoffe das bleit so.
